### PR TITLE
[12.0] [MIG] account_invoice_refund_link

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Link refund invoice with original
=================================

This module shows the links between refunds and their original invoices in the invoice form and also keep track of refund lines and their original invoice lines.

### CODE REVIEW

File: account_invoice_refund_link/views/account_invoice_view.xml [hunk 1]
```diff
@@ -5,82 +5,82 @@
...
-                   attrs="{'invisible': [('refund_invoice_id', '=', False)]}"
-            />
-            <field name="refund_reason"
-                   readonly="1"
-                   attrs="{'invisible': [('refund_reason', '=', False)]}"
-            />
-        </xpath>
-    </field>
-</record>
+    <record id="view_customer_account_invoice_add_refunds_details_form" model="ir.ui.view">
```
Comment (pedrobaeza) [inline account_invoice_refund_link/views/account_invoice_view.xml]: Please don't change this only for esthetic reasons, as it creates a great diff with no need.

**Review by pedrobaeza — COMMENTED** [review]
Squash together adjacent "OCA Transbot..." commits.

File: account_invoice_refund_link/views/account_invoice_view.xml (hunk 1 above)
Comment (Nikul-Chaudhary) [reply account_invoice_refund_link/views/account_invoice_view.xml]: > Please don't change this only for esthetic reasons, as it creates a great diff with no need.

@pedrobaeza Yes, But it's look ugly, so I changed.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Squashed

File: account_invoice_refund_link/__manifest__.py [hunk 2]
```diff
@@ -6,9 +6,9 @@
 {
     "name": "Link refund invoice with original",
     "summary": "Link refund invoice with its original invoice",
-    "version": "11.0.2.0.0",
+    "version": "12.0.2.0.0",
```
Comment (pedrobaeza) [inline account_invoice_refund_link/__manifest__.py]: 12.0.1.0.0

File: account_invoice_refund_link/models/account_invoice.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    refund_reason = fields.Text(string="Refund reason")
+    refund_reason = fields.Text("Refund reason")
```
Comment (pedrobaeza) [inline account_invoice_refund_link/models/account_invoice.py]: Don't remove keyword attribute. They are always more flexible and it's already there. Why increasing the diff?

File: account_invoice_refund_link/views/account_invoice_view.xml (hunk 1 above)
Comment (pedrobaeza) [reply account_invoice_refund_link/views/account_invoice_view.xml]: Please don't

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

**Review by misern2 — APPROVED** [review]

**Pull request merged**